second.ino

```diff
@@ -419,7 +331,7 @@ int calcPID(int error, int previousError)
 }
 
 //==============set motor speed based on PID error==============//
-void setLeftRightSpeed(int PIDvalue, int error)
+void setLeftRightSpeed(int PIDvalue, int error) // This virable int error is defined as globale above :)
```

Comment:
This is 100% working for some conflicted reasons, so be cautious when setting it as a local variable if you decided to.